pyaerocom/aeroval/compute_fairmode_stats.py

```diff
@@ -0,0 +1,98 @@
+"""
+Functions for computing the FAIRMODE statistics
+"""
+
+from math import sqrt, isclose
+from typing import Tuple
+
+
+def _get_RMSU(mean: float, std: float, species: str) -> Tuple[float, float, float, float]:
```

Comment:
Why `_get_RMSU` instead of `_RMSU`?
I do not see any web request or any kind of `get` action.

Also, a docstring about what the acronym stands for would be useful for someone unfamiliar with the FAIRMODE indicators, like me

Reply:
Addressed